rules/apps.libsonnet

```diff
@@ -21,9 +21,9 @@
             // quantile_over_time(...) which would otherwise not be possible.
             record: 'node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate',
             expr: |||
-              sum by (namespace, pod, container) (
+              sum by (cluster, namespace, pod, container) (
```

Comment:
We can't assume this label, but we have configuration for it in the `$._config.clusterLabel` variable which can be used here.

Reply:
Thank you for pointing out.

Next commit is using the configuration.